### PR TITLE
Warn about withdrawing the whole balance

### PR DESCRIPTION
**Pull request by sindresorhus**
Since it doesn't seem like #302 will be fixed anytime soon and lots of users are encountering it, we should make the issue clear and prevent it from happening.

screen shot 2018-09-25 at 15 44 55

@lukechilds Is the text correct?

### CODE REVIEW

**Comment by lukechilds** [comment]
The issue is a bit more complex than that. But without going to far down the rabbit hole, yeah that's the general idea.

We can actually resolve this with most alts and ACs as they have a hardcoded txfee so we can just reduce this from the amount. However I was pretty stubborn about not implementing this as it would mean we would need completely different withdrawal logic for different currencies.

I'd much rather we find the time to put the work in to get proper withdrawals working with fees and use the same code for all currencies.

Off the top of my head, BTC, ETH and all ERC20 tokens use dynamic fees so will require a fair bit of work to integrate properly (probably not possible in a user friendly way with current mm). However, all other currencies would work fine using their hardcoded txfee and subtracting that from the available amount.

Also, I'm not convinced this is the only cause of the error we get from withdrawals. We seem to get lots of errors in other scenarios too, unrelated to large balances but the error response gives us no clue as to what this is, they are all the same.

**TL;DR Yeah, I'd say we should merge this to hopefully reduce the confusion and amount of support requests we get regarding it, but it's not really explaining the full picture.**
